authentication/models.py

```diff
@@ -42,14 +41,15 @@ def save_user_profile(instance, **_):
 
 class UserBank(models.Model):
     """
-    Contains all the user's bank access data
+    Contains all the user's bank access data (via plaid)
     """
     user = models.ForeignKey(
         User,
         on_delete=models.CASCADE
     )
     item_id = models.CharField(max_length=1000)
     access_token = models.CharField(max_length=1000)
+    institution_name = models.CharField(max_length=1000)
```

Comment:
That's hella long. We can probably cut all the char lengths down. Plaid should provide specs?

Reply:
I don't think there's any performance difference in terms of postgresql; this is just (required) validation. I checked plaid and unfortunately there's nothing there.